_schemas/1.0/schema_update_resource.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
```suggestion
  "$schema": "https://json-schema.org/draft/2020-12/schema",,
```

Reply:
can we switch to this latest schema specs or v1.0 will have to stay with draft-7?

Reply:
IMHO we can switch to latest version.

Reply:
done